azure-devops/azext_devops/dev/boards/relations.py

```diff
@@ -72,17 +72,21 @@ def remove_relation(id, relation_type, target_id, organization=None, detect=None
 
     main_work_item = client.get_work_item(id, expand='All')
 
-    for target_work_item_id in target_work_item_ids:
-        target_work_item = client.get_work_item(target_work_item_id, expand='All')
-        target_work_item_url = target_work_item.url
+    if main_work_item.relations is not None:
```

Comment:
nit: relations is a list right? 
if not relations: is a better way to check none or empty.

Reply:
just a not should handle every case, changed